gapis/api/gles/state_builder.go

```diff
@@ -201,6 +201,10 @@ func (sb *stateBuilder) contextObject(ctx context.Context, handle EGLContext, c
 	if !c.Other().Initialized() {
 		return
 	}
+	if c.Other().Destroyed() {
```

Comment:
Not too sure what the point is in creating a context just to delete it again.
Seems a bit excessive for the correctness argument.

Reply:
Done.